blog/2024-09-24-python313.md

```diff
@@ -0,0 +1,56 @@
+---
+authors:
+  - isuruf
+tags: [infrastructure]
+---
+
+# Python 3.13 Release candidate builds on conda-forge
+
+conda-forge now supports python 3.13 release candidates on conda.
+You can create a new environment with python 3.13 by running the
+command:
+
+    conda create -n py313 python=3.13 -c conda-forge/label/python_rc -c conda-forge
+
+This will create a new environment with python 3.13 with global
+interpreter lock (GIL) enabled. A migration is underway that builds
+python extensions like numpy and scipy conda packages. The migration
+is 50% complete at the time of writing. Once the first release of
+python 3.13.0 is available the `python_rc` label will not be needed
+anymore.
+
+New in this python release is the python-freethreading build which
+removes the GIL and enables free threading. To install a freethreading
+build, you can do:
+
+    conda create -n py313 python=3.13 python-freethreading -c conda-forge/label/python_rc -c conda-forge
+
+Analogous to this package we also have a meta-package to explicitly
+install the GIL variant:
```

Comment:
Is this the variant with our without GIL?

Reply:
with the GIL